packages/trpc/server/middlewares/sessionMiddleware.ts

```diff
@@ -75,6 +76,12 @@ export async function getUserFromSession(ctx: TRPCContextInner, session: Maybe<S
         select: {
           slug: true,
           metadata: true,
+          members: {
+            where: {
+              userId: session.user.id,
+              OR: [{ role: MembershipRole.ADMIN }, { role: MembershipRole.OWNER }],
+            },
+          },
```

Comment:
@sean-brydon filtered in the Prisma query

Reply:
Do we want full member data here?